ignite/contrib/handlers/wandb_logger.py

```diff
@@ -301,6 +302,7 @@ def score_function(engine):
                 Events.COMPLETED, model_checkpoint, {'model': model})
     """
 
+    @one_rank_only()
```

Comment:
Maybe we need also to decorate 
```python
    def __getattr__(self, attr):
        return getattr(self._wandb, attr)
```
as it can be potentially used

Reply:
Done. The `wandb` version seems better.